src/movies.js

```diff
@@ -1,16 +1,93 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+   function getAllDirectors (movies){
+       const arrayDirectors = movies.map(movies => movies.director)  
+           return arrayDirectors
+   }
```

Comment:
Si refactorizamos un poco quedaría:
```suggestion
   function getAllDirectors (movies){
      return movies.map(movies => movies.director)  
   }
```